auth/server/src/main/java/com/seagates3/authserver/AuthServerConfig.java

```diff
@@ -131,19 +133,43 @@ public static void logConfigProps() {
      * @throws GeneralSecurityException
      */
     public static void loadCredentials() throws GeneralSecurityException, Exception {
-
-        Properties authServerConfig = AuthServerConfig.authServerConfig;
-        String encryptedPasswd = authServerConfig.getProperty("ldapLoginPW");
-        Path keyStoreFilePath = getKeyStorePath();
-        PrivateKey privateKey = JKSUtil.getPrivateKeyFromJKS(
-                                keyStoreFilePath.toString(), getCertAlias(),
-                                getKeyStorePassword());
-        if (privateKey == null) {
-             throw new GeneralSecurityException("Failed to find Private Key ["
-                + keyStoreFilePath + "].");
-        }
-        ldapPasswd = RSAEncryptDecryptUtil.decrypt(encryptedPasswd,
-                     privateKey);
+       logger = LoggerFactory.getLogger(AuthServerConfig.class.getName());
+       String keystorePasswd = null;
+       String encryptedPasswd = authServerConfig.getProperty("ldapLoginPW");
+       Path keyStoreFilePath = getKeyStorePath();
+       try {
+         String cmd = authServerConfig.getProperty("cortexSecCmd");
+         logger.info("Command is : " + cmd);
+         Process cortxsec = Runtime.getRuntime().exec(cmd);
+         int exitVal = cortxsec.waitFor();
+         if (exitVal != 0) {
+           throw new IOException("Process exited with error code");
+         }
+         BufferedReader reader = new BufferedReader(
+             new InputStreamReader(cortxsec.getInputStream()));
+         String line;
+         if ((line = reader.readLine()) != null) {
+           keystorePasswd = line;
+         } else {
+           throw new IOException("Input Stream null");
+         }
+       }
+       catch (IOException e) {
+         // logger.info(" Default ldap load credentials due to error - " +
+         // e.getMessage());
+         logger.info(" Default ldap load credentials due to error - " +
```

Comment:
Default ldap load credentials **failed**, error - 

I am not sure if authserver has other logging levels, but I see this log is better suited for error log level.

Reply:
Same as above.. Not yet for review.